### ISSUE TITLE
`volta list --default` doesn't show defaults

### ISSUE THREAD

**Issue by PSalant726** [volta-cli/volta#922]
# Expected
```
$ volta install node
success: installed and set node@14.15.4 as default

$ volta list --default node

    v14.15.4
```
# Actual
```
$ volta install node
success: installed and set node@14.15.4 as default

$ volta list node
⚡️ Node runtimes in your toolchain:

    v14.15.4

$ volta list --default node
⚡️ No Node runtimes installed!

    You can install a runtime by running `volta install node`. See `volta help install` for
    details and more options.
```

Volta 1.0.1
macOS 11.1

**Comment by charlespierce** [comment]
Hi @PSalant726, thanks for reporting this! I was already planning to take a look at the `volta list` output, so I'll make sure to include that in my investigation.

**Comment by charlespierce** [comment]
Just confirmed this is fixed in #778 (which I had left outstanding for some time 🤦 ). Merged that so it should be in the next release!
